shuttle/simple_html_dom.php

```diff
@@ -0,0 +1,1742 @@
+<?php
```

Comment:
yea no, please don't add dependencies directly to our git repo but instead use composer.